querybook/tests/test_lib/test_query_analysis/test_create_table/test_helper.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+from lib.query_analysis.create_table.helper import is_custom_column_type
+from lib.table_upload.common import UploadTableColumnType
+
+
+class IsCustomColumnTypeTestCase(TestCase):
+    def test_included_types(self):
+        self.assertFalse(is_custom_column_type(UploadTableColumnType.INTEGER))
```

Comment:
Should these take value of the enum memeber: e.g. `UploadTableColumnType.INTEGER.value`